sql/core/src/test/scala/org/apache/spark/sql/UserDefinedTypeSuite.scala

```diff
@@ -31,48 +30,7 @@ import org.apache.spark.sql.types._
 @BeanInfo
 private[sql] case class MyLabeledPoint(
   @BeanProperty label: Double,
-  @BeanProperty features: UDT.MyDenseVector)
-
-// Wrapped in an object to check Scala compatibility. See SPARK-13929
-object UDT {
```

Comment:
why do we change it?

Reply:
That's because we use `MyDenseVectorUDT` in `UnsafeRowConverterSuite.scala` for unit tests. (`MyDenseVectorUDT` is located in the `core`, but `UnsafeRowConverterSuite` located in the `catalyst`).